[azure]: Handle Azure DNS refs #112

zelima opened the pull request:
This assumes tha Azure DNS Zone is already created. 

_Note:_ I have tested this and I see the DNS records are created. But have not actually teste the app is running on create DNS record, like data.viderum.net (still trying to figure it out)

zelima committed:
[azure]: Handle Azure DNS refs #112

This assumes tha Azure DNS Zone is already created.

zelima committed:
Execute command from correct place

zelima committed:
Give time loadbalancer IP to appear

zelima committed:
wrap cluster initilaization commands into 1 script

akariv reviewed (approved):
Looks good!

Seems that you need to chmod +x init_cluster.sh though :)

zelima committed:
Set azure as DNS provider and create DNS zone with terraform

zelima committed:
Docs for provisioning Ckan-Cloud with cco on azure

zelima committed:
[Terraform] Allos creating RG and DNS zone or passing their names

zelima committed:
Let traefik creating letsencript certificates

- Back to azure-disk az it has correct (readWriteOnce) filesystem mode
- Pass SP credentials to cco
- Use SP credentials to create certificates (for traefik)
- Use updated version of ckan helm charts (one that uses ReadWriteOnce mode)
- Update Docs to include tenant ID and Subscription ID as it's required by traefik

zelima committed:
Remove sku as it is depreiciated now. And pass location and cluster names

zelima committed:
use random_string to make sure its lowercase

zelima committed:
make sure cluster suffics is lowercase

zelima committed:
include scu_nmae

zelima committed:
explicitly copy kubeconfig in ~/.kube/config

zelima committed:
create ~/.kube/ directory
